src/main/java/net/mcreator/ui/action/accelerators/Accelerator.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2022, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.action.accelerators;
+
+import org.apache.commons.lang3.StringUtils;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.KeyEvent;
+
+/**
+ * <p>A class used to save the default {@link KeyStroke} and the current one. Each accelerator is linked to a {@link net.mcreator.ui.action.BasicAction}
+ * in its constructor.</p>
+ */
+public class Accelerator {
+
+	public static final int CTRL = Toolkit.getDefaultToolkit().getMenuShortcutKeyMaskEx();
+	public static final int CTRL_SHIFT = CTRL | KeyEvent.SHIFT_DOWN_MASK;
+	public static final int CTRL_ALT = CTRL | KeyEvent.ALT_DOWN_MASK;

```

Comment:
What about CTRL_ALT_SHIFT? Shouldn't it be here too?

Reply:
I did not add it because no default accelerator uses it, so it would be useless. It can be added when a default accelerator will need it.